nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/pom.xml

```diff
@@ -628,8 +634,13 @@
                                 js/d3/**/*,
                                 js/codemirror/**/*,
                                 js/jquery/**/*,
+                                js/**/**/*,
```

Comment:
This was for the new angular libs, I will update it to be more specific: 

js/angular/**/*